### ISSUE TITLE
Test failure in nix package build.

### ISSUE THREAD

**Issue by alunduil** [alunduil/zfs-replicate#35]
```
running install tests
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-4.2.1, py-1.7.0, pluggy-0.8.1
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/build/zfs-replicate-1.1.3/.hypothesis/examples')
rootdir: /build/zfs-replicate-1.1.3, inifile:
plugins: hypothesis-4.7.3, cov-2.6.1
collected 11 items

zfs/replicate/list.py .                                                  [  9%]
zfs_test/replicate_test/list_test.py .....                               [ 54%]
zfs_test/replicate_test/optional_test.py ..                              [ 72%]
zfs_test/replicate_test/cli_test/main_test.py F                          [ 81%]
zfs_test/replicate_test/snapshot_test/list_test.py ..                    [100%]

=================================== FAILURES ===================================
_______________________ test_invokes_without_stacktrace ________________________

    def test_invokes_without_stacktrace() -> None:
        """zfs-replicate -l alunduil -i mypy.ini example.com bogus bogus => No stacktrace"""

        runner = CliRunner()
        result = runner.invoke(main, ["-l", "alunduil", "-i", "mypy.ini", "example.com", "bogus", "bogus"])
>       assert isinstance(result.exception, SystemExit), "Expected normal exit."
E       AssertionError: Expected normal exit.
E       assert False
E        +  where False = isinstance(FileNotFoundError(2, "No such file or directory: '/usr/bin/env'"), SystemExit)
E        +    where FileNotFoundError(2, "No such file or directory: '/usr/bin/env'") = <Result FileNotFoundError(2, "No such file or directory: '/usr/bin/env'")>.exception

zfs_test/replicate_test/cli_test/main_test.py:11: AssertionError
=============================== warnings summary ===============================
/nix/store/6l2gzgn4h8zyrbc051m8xv393s5gxzd9-python3.6-stringcase-1.2.0/lib/python3.6/site-packages/stringcase.py:247
  /nix/store/6l2gzgn4h8zyrbc051m8xv393s5gxzd9-python3.6-stringcase-1.2.0/lib/python3.6/site-packages/stringcase.py:247: DeprecationWarning: invalid escape sequence \W
    return re.sub("\W+", "", string)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
=============== 1 failed, 10 passed, 1 warnings in 2.46 seconds ================
```

**Comment by alunduil** [comment]
I'm moving this to the alunduil-nixos instead as the issue is in the build environment more than this script.

**Comment by alunduil** [comment]
I'm cross linking the old nix build issues to their modern successors. I closed this one deliberately at the time, since the failure was in the build environment rather than in the script, and it's still marked invalid, so I'm leaving the state alone rather than reworking it into a duplicate. For anyone arriving here from a search: the `/usr/bin/env` hardcoding behind that failure is addressed in #432, which landed in #474, and the unpinned nix expression that made these builds fragile in the first place is addressed in #430.
